modules/apps/data-engine/data-engine-taglib/src/main/resources/META-INF/resources/data_layout_builder/js/drag-and-drop/dragTypes.es.js

```diff
@@ -15,5 +15,6 @@
 export const DRAG_FIELD_TYPE_MOVE = 'fieldType:move';
 
 export const DRAG_DATA_DEFINITION_FIELD_ADD = 'dataDefinitionField:add';
+export const DRAG_ELEMENT_SET_ADD = 'elementSet:add';
```

Comment:
Well, although we use these DragTypes here on the Data Engine, it is a little strange... but for this case, I prefer that we keep a constant in the `useDrop` hook.